### PR TITLE
Rename py2-dablooms to dablooms. Remove dablooms python binding. 

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by gartung** [comment]
please test

**Comment by gartung** [comment]
abort

**Comment by gartung** [comment]
test

**Comment by gartung** [comment]
abort

**Comment by gartung** [comment]
test

**Comment by gartung** [comment]
abort

File: bloom-filter-toolfile.spec [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/bloom-filter.xml
+<tool name="bloom-filter" version="@TOOL_VERSION@">
+  <client>
+    <environment name="BF_BASE"   default="@TOOL_ROOT@"/>
```
Comment (smuzaffar) [inline bloom-filter-toolfile.spec]: please use BLOOM_FILTER instead of BF

File: python_tools.spec [hunk 2]
```diff
@@ -137,7 +137,6 @@ Requires: py2-matplotlib
 Requires: py2-numpy-toolfile
 Requires: py2-sqlalchemy
 Requires: py2-pygithub
-Requires: py2-dablooms-toolfile
```
Comment (smuzaffar) [inline python_tools.spec:140]: we need a cmssw PR to go with this change. It will fail PR tests and IBs due to missing dablooms

**Comment by gartung** [comment]
I am working on an PR for StaticAnalyzers.

File: python_tools.spec (hunk 2 above)
Comment (gartung) [reply python_tools.spec:140]: I am working on a coordinated change to StaticAnalyzers.

File: dablooms-toolfile.spec [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+%install
+
+mkdir -p %{i}/etc/scram.d
+cat << \EOF_TOOLFILE >%{i}/etc/scram.d/dablooms.xml
+<tool name="dablooms" version="@TOOL_VERSION@">
+  <lib name="dablooms"/>
+  <client>
+    <environment name="DABLOOMS_BASE" default="@TOOL_ROOT@"/>
+    <environment name="INCLUDE" default="$DABLOOMS_BASE/include"/>
```
Comment (smuzaffar) [inline dablooms-toolfile.spec:17]: @gartung , looks like you are missing LIBDIR here?

Comment (gartung) [reply dablooms-toolfile.spec:17]: Good catch. I must have picked it up because py2-dablooms was configured and linked libdablooms into external.

**Comment by gartung** [comment]
test with https://github.com/cms-sw/cmssw/pull/28315

File: dablooms-toolfile.spec (hunk 3 above)
Comment (gartung) [reply dablooms-toolfile.spec:17]: And I didn't reconfigure externals when I did scram setup dablooms.

Comment (gartung) [reply dablooms-toolfile.spec:17]: The library was picked up from CMSSW_RELEASE_BASE externals.

**Comment by smuzaffar** [comment]
+externals

**Comment by fabiocos** [comment]
+1

**Pull request merged**